src/clojure/nrepl/middleware/caught.clj

```diff
@@ -0,0 +1,97 @@
+(ns nrepl.middleware.caught
+  "Support for a hook for conveying errors interactively, akin to the `:caught`
+  option of `clojure.main/repl`. See the docstring of `wrap-caught` and the
+  Evaluation Errors section of the Middleware documentation for more
+  information."
+  {:author "Michael Griffiths"}
```

Comment:
You can also add `:added “0.6”` here as a bit of extra meta info.

Reply:
Done.